about/index.html

```diff
@@ -67,7 +67,7 @@
       <!-- End Content -->
         <!-- Begin Footer -->
         <div class="footer">
-          <p>&copy; 2011-2016 Hack the Future</p>
```

Comment:
For next time, it might be good to replace the hyphen with an en dash, which is used to specify a range.
2011–2018 instead of
2011-2018